security-updates/index.rst

```diff
@@ -57,6 +57,24 @@ Ubuntu Pro is a subscription that provides access to several security-focused
 features and services. You can read more about it in the `Ubuntu Pro
 documentation </pro/services-overview/>`_.
 
+Update Notifications
+====================
+
+Ubuntu users receive notifications when new updates are available as part of
+the ``update-manager`` package (which is known as ``Software & Updates`` in the
+desktop menu). Users can use it to configure the notifications, manage updates,
+as well as manage automatic update settings (with more information in the
+"Automatic security updates" section below)
```

Comment:
Would you also be able to have the "Automatic security updates" text link to the section? You should be able to accomplish this by giving the section a label and then linking to said label.